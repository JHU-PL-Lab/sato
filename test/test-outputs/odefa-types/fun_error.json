[
  {
    "err_errors": [
      [
        "Error_match",
        {
          "err_match_aliases": [ "zero" ],
          "err_match_val": "0",
          "err_match_expected": "fun",
          "err_match_actual": "int"
        }
      ]
    ],
    "err_input_seq": [],
    "err_location": "ans = zero zero",
    "err_steps": 38
  }
]