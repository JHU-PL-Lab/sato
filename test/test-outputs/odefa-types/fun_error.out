Type errors for:
- Input sequence  : []
- Found at clause : ans = zero zero
--------------------
* Value    : zero = 0
* Expected : fun
* Actual   : int
1 errors found. No further control flows exist.
